std/net/curl.d

```diff
@@ -203,9 +203,7 @@ version (unittest)
             }
             catch (Throwable e)
             {
-                import core.stdc.stdlib : exit, EXIT_FAILURE;
-                stderr.writeln(e);
-                exit(EXIT_FAILURE); // Bugzilla 7018
+                stderr.writeln(e);  // Bugzilla 7018
```

Comment:
unrelated?

Reply:
Yes, looks like it's a leftover or something from the previous master commit.  What's the easiest way for me to fix these?

Reply:
@benjones You can amend the commit and force push it.